src/components/FAQ/styles.module.css

```diff
@@ -0,0 +1,17 @@
+.faq-wrapper {
+    font-family: Helvetica;
+}
+
+.faq-question {
+    color: var(--tp-bright-purple);
+    font-weight: bold;
+    font-size: 2.9rem;
```

Comment:
Can we round the sizes?

`font-size: 3rem` etc.

Reply:
We can, but it will deviate from the layout as it will not be pixel perfect.

Would you like me to update it anyways?
- [ ] Agree with the terms and conditions

Reply:
I'll try to make sure the designs are rounded in the future. I think changing these is okay.

Reply:
Also, I don't think that you're a member of our Slack. Would you like to join?

Reply:
> I'll try to make sure the designs are rounded in the future. I think changing these is okay.

Done.

> Also, I don't think that you're a member of our Slack. Would you like to join?

I'd be glad to.
Send an invite to `forsureitsme@gmail.com`.

Reply:
Awesome! Sent!